src/gui/plugins/visualization_capabilities/VisualizationCapabilities.cc

```diff
@@ -99,6 +101,13 @@ namespace ignition::gazebo
     public: std::vector<ignition::gazebo::Entity>
       FindChildLinks(const ignition::gazebo::Entity &_entity);
 
+    /// \brief Helper function to get all children of an entity that have a
+    /// pose.
+    /// \param[in] _entity Entity to find children
+    /// \return Vector of children found for the parent entity
+    public: std::unordered_set<ignition::gazebo::Entity>
+      FindChildFrames(const ignition::gazebo::Entity &_entity);
```

Comment:
Yeah good point. It will need to be cleaned up and tested more. I ticketed an issue because there are other functions in this plugin in the same situation:

* https://github.com/ignitionrobotics/ign-gazebo/issues/1242